spec/lib/va_profile/v2/contact_information/service_spec.rb

```diff
@@ -38,16 +38,34 @@
       # end
 
       it 'has a bad address' do
-        VCR.use_cassette('va_profile/v2/contact_information/person_full', VCR::MATCH_EVERYTHING) do
+        VCR.use_cassette('va_profile/v2/contact_information/person', VCR::MATCH_EVERYTHING) do
           response = subject.get_person
           expect(response.person.addresses[0].bad_address).to eq(nil)
         end
       end
     end
 
-    context 'when not successful' do
-      let(:user) { build(:user, :error) }
+    context 'when person response has no body data' do
+      it 'returns 200' do
+        VCR.use_cassette('va_profile/v2/contact_information/person_without_data', VCR::MATCH_EVERYTHING) do
+          response = subject.get_person
+          expect(response).to be_ok
+          expect(response.person).to be_a(VAProfile::Models::V2::Person)
+        end
+      end
+    end
+  end
 
+  describe '#get_person error' do
+    let(:user) { build(:user, :loa3) }
```

Comment:
you can do:
`build(:user, :loa3, vet360_id: '6767671', idme_uuid: nil)` to put the attributes you're setting here instead of in the `before do` stubbing

Reply:
I will also make a note to redo this for the refactor.